x-pack/legacy/plugins/siem/server/graphql/scalar_to_any/resolvers.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { GraphQLScalarType, Kind } from 'graphql';
+
+/*
+ *  serialize: gets invoked when serializing the result to send it back to a client.
+ *
+ *  parseValue: gets invoked to parse client input that was passed through variables.
+ *
+ *  parseLiteral: gets invoked to parse client input that was passed inline in the query.
+ */
+
+export const toAnyScalar = new GraphQLScalarType({
+  name: 'Any',
+  description: 'Represents any type',
+  serialize(value): unknown {
+    if (value == null) {
+      return null;
+    } else if (typeof value === 'string' && value.charAt(0) === '{') {
+      return JSON.parse(value);
```

Comment:
I'd change this to have a try catch and then test if the resulting thing is an object like so:

```ts
try {
     const maybeObj = JSON.parse(value);
    if (isObject(maybeOb)) {
      return maybeObj;
    } else {
      return value
    }
   } catch (e) {
      return value;
    }
```